av1an-core/src/lib.rs

```diff
@@ -129,11 +137,19 @@ impl<P: AsRef<Path> + Into<PathBuf>> From<P> for Input {
   }
 }
 
+#[derive(Debug, Deserialize, Serialize, Clone, Copy)]
+struct DoneChunk {
+  frames: usize,
+  // Currently an Option for backwards compatibility,
+  // since this field was added recently.
+  size_kb: Option<u32>,
```

Comment:
I don't think this would actually work to maintain backwards compatibility, as serde serializes `Option::None` in json in this case as `size_kb: null`, so the old way of serializing it would not work. I also just tried it on an older temp folder, and this is the output I got:

```
Error: invalid type: integer `147`, expected struct DoneChunk at line 1 column 34
```

Theoretically we still *could* maintain backwards compatibility by trying to parse the json both ways, but I think that gets ugly pretty fast. If we are going to switch away from json eventually anyway, I think it would be ok to just drop backwards compatibility for this, and perhaps we could land this in a 'nightly' or 'experimental' branch for now as discussed in discord before. What do you think?